### ISSUE TITLE
[Category] Short description

### ISSUE THREAD

**Issue by Raenora6** [sp614x/optifine#5821]
## Description of Issue
Random strob lights appear periodically and at random though more common at night.

## Steps to Reproduce
nothing was walking back and forth to different areas

## OptiFine Version
preview ver. 1.17 OF

## Installation Method
Standalone

## Other Installed Mods
Currently have clarity and Winthor+medival resource packs downloaded though only using Win+med.

## F3 Debug Screenshot
![2021-06-16_17 02 15](https://user-images.githubusercontent.com/86044026/122352573-dbde3300-cf14-11eb-90a5-57e895cdeeb5.png)

## Prior Testing
<!-- If you've installed standalone:
- Does this happen in Vanilla without OptiFine?
no it does not happen without OF

**Comment by Whitelisted1** [comment]
Does this still happen with the latest version of Optifine pre? Just seeing if this issue is still relevant

**Comment by Raenora6** [comment]
> Does this still happen with the latest version of Optifine pre? Just seeing if this issue is still relevant

yes it still happens from time to time though I notice its not as often as it used to be! Sorry for the late reply I don't normally check my emails! I am in the discord server for optifine though!
